### ISSUE TITLE
Is there any way to obtain datapoints which are close to my incoming new data point after the model training

### ISSUE THREAD

**Issue by bharathwajan** [JustGlowing/minisom#150]
First I want to appreciate for the great work . And my question is
1. consider that I had a test data(a single record) which needs to be clustered and SOM done the job
2. after this clustering I wanna find other points which are closer to my test data or is there any function to get the data points which falls under the same cluster
Thanks in Advance

**Comment by JustGlowing** [comment]
hi, your question is a tad hard to follow but I'll attempt an answer.

The method `winner` tells you the cell with weights that are similar to your input data. To find which points are mapped in that cell, hence close to what you want, you can follow this tutorial https://github.com/JustGlowing/minisom/blob/master/examples/Clustering.ipynb

**Comment by bharathwajan** [comment]
Is there any way to identify the points which falls in a particular cluster
say that ,I want identify which of my training data falls in cluster 1 ?

**Comment by JustGlowing** [comment]
The example linked in my previous comment shows you how to get the cluster index for each data point.
